packages/core/src/libraries/hook.ts

```diff
@@ -15,6 +15,9 @@ import { LogEntry } from '#src/middleware/koa-audit-log.js';
 import type Queries from '#src/tenants/Queries.js';
 import { consoleLog } from '#src/utils/console.js';
 
+// Note: the retry limit is fixed to 3 for now.
+const hookRetryLimit = 3;
```

Comment:
Defined `hookRetryLimit` here and a `defaultRetries` in the alteration script, maybe we can unify these constants so that we won't miss them?

Reply:
Values can not be shared between alteration scripts and other package codes but it's good to make them have the same name.